__tests__/settings.test.js

```diff
@@ -0,0 +1,27 @@
+import errors from '../src/errors';
+import settingsFixture from './__fixtures__/settings-fixture';
+import TestUtils from './test-utils';
+
+test('wallet lib settings', async () => {
+  const settings = jest.requireActual('../src/settings');
+  expect(settings.getConfig).toThrow(errors.UnavailableConfigError);
+
+  const config = settingsFixture._getDefaultConfig();
+
+  jest.doMock('../src/config', () => ({
+    __esModule: true,
+    default: config,
+  }));
+
+  await settings.setupConfig();
+  await expect(settings.setupConfig()).rejects.toThrow('Cannot setup the configuration twice');
+
+  expect(settings.getConfig).not.toThrow();
```

Comment:
I suggest doing a double check, maybe throwing an error arbitrarily and verifying if the test still passes. I say this because the [Jest documentation](https://jestjs.io/docs/expect#tothrowerror) indicates that wrapping in a function is mandatory.